install/README.md

```diff
@@ -8,28 +8,61 @@ To get started with Knative, you need a Kubernetes cluster. If you aren't
 sure which Kubernetes platform is right for you, see
 [Picking the Right Solution](https://kubernetes.io/docs/setup/pick-right-solution/).
 
-We provide information for installing Knative on
-[Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/docs/), [IBM Cloud Kubernetes Service](https://www.ibm.com/cloud/container-service), [Azure Kubernetes Service](https://docs.microsoft.com/en-us/azure/aks/), [Minikube](https://kubernetes.io/docs/setup/minikube/), [OpenShift](https://github.com/openshift/origin) and [Pivotal Container Service](https://pivotal.io/platform/pivotal-container-service) clusters.
+We provide information for installing Knative on multiple different platforms.
 
 ## Installing Knative
 
+There are several options when installing Knative:
+
+* **Bundle install** -- Comes with the default versions of all Knative
+  components. Quickest option for setup.
+
+* **Limited install** -- Installs a subset of Knative components.
+
+* **Custom install** -- Takes longer, but allows you to choose exactly which
+  components to install.
+
+For new users, we recommend the bundle install to get you up and running
+quickly.
+
+### Install guides
+
 Follow these step-by-step guides for setting up Kubernetes and installing
-Knative components on the following platforms:
+Knative components.
 
+**Bundle install guides**
+The guides below show you how to create a Kubernetes cluster with the right
+specs for Knative on your platform of choice, then walk through installing all
+available Knative components.
 * [Knative Install on Azure Kubernetes Service](Knative-with-AKS.md)
 * [Knative Install on Gardener](Knative-with-Gardener.md)
 * [Knative Install on Google Kubernetes Engine](Knative-with-GKE.md)
 * [Knative Install on IBM Cloud Kubernetes Service](Knative-with-IKS.md)
-* [Knative Install on Minikube](Knative-with-Minikube.md)
-* [Knative Install on OpenShift](Knative-with-OpenShift.md)
-* [Knative Install on Minishift](Knative-with-Minishift.md)
 * [Knative Install on Pivotal Container Service](Knative-with-PKS.md)
 
-If you already have a Kubernetes cluster you're comfortable installing
-*alpha* software on, use the following instructions:
+If you already have a Kubernetes cluster you're comfortable installing *alpha*
+software on, use the following guide to install all Knative components:
 
 * [Knative Install on any Kubernetes](Knative-with-any-k8s.md)
 
+**Limited install guides**
```

Comment:
Not sure how i feel about this categorization since these are the same bundles as above but offer either partial metrics or no observability all together (more comments about this below).  

I think the difficulty we are facing is the unbalance and exclusion of certain configurations of install packages (ie Serving, Build, and Stackdriver(vs ELK) ...or.... Serving, Build, Eventing w/ & w/o ELK or Stackdriver). -> We are definitely missing a Serving & Eventing type package since the [Eventing install steps](https://github.com/knative/docs/tree/master/eventing#installation) mention that Serving is 'currently' required for Eventing.

If there was a clear set of packages that had intuitive and consistent names, then there could be "a choose your observability equivalent" (like an easier pre-custom option/category) bundle alongside the "(install everything) Bundle installs"?

Reply:
how about bundle vs individual component install guides?